plugins/scaffolder/src/types.ts

```diff
@@ -64,3 +64,14 @@ export type ListActionsResponse = Array<{
     output?: JSONSchema;
   };
 }>;
+
+type OutputLink = {
+  url: string;
+  title?: string;
+  icon?: string;
+};
+
+export type TaskOutput = {
+  entityRef?: string;
+  links?: OutputLink[];
+} & { [key in string]: string };
```

Comment:
This type is wrong (and is causing the build failure). I don't yet know how to fix it.

Reply:
This will work but will mean typing remaining props as `any`.

```ts
export type TaskOutput = {
  entityRef?: string;
  remoteUrl?: string;
  links?: OutputLink[];
  [key: string]: any;
};
```